app/src/main/java/io/nwhacks/nfc/ApplicantInfo.java

```diff
@@ -0,0 +1,22 @@
+package io.nwhacks.nfc;
+
+import java.util.Map;
+
+/**
+ * Created by rice on 11/30/17.
+ */
+
+public class ApplicantInfo {
+    public String firstName;
+    public String lastName;
+    public String email;
+    public Map<String, Integer> events;
+    public static Map<String, String> applicantMap;
+
+    static {
```

Comment:
I get the an error when trying to read/write a tag, here's the root error:

```
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object java.util.Map.put(java.lang.Object, java.lang.Object)' on a null object reference
        at io.nwhacks.nfc.ApplicantInfo.<clinit>(ApplicantInfo.java:17)
```

I'm not sure if it's working on your end, but it looks like you have to instantiate the `applicantMap` Map first. Seems like there are a couple ways to do this, here's a sensible solution I found: https://stackoverflow.com/a/509016

Reply:
I fixed this by using an ImutableMap to create the map statically.

Reply:
It's working!